src/components/UploadForm.js

```diff
@@ -0,0 +1,39 @@
+import React, { useState } from 'react'
+import ProgressBar from './ProgressBar';
+import "../styles/upload-form.css"
+
+const UploadForm = () => {
+  
+
+    const [ file, setFile ] = useState(null);
+    const [ error, setError ] = useState(null);
+    
+    const types = ['image/png', 'image/jpeg']
+
+    const changeHandler = (e) => { 
+        let selected = e.target.files[0];
+        console.log(selected)
+        if(selected && types.includes(selected.type)) { 
+            setFile(selected)
+            setError('')
+        } else {
+            setFile(null)
+            setError('Please select an acceptable image type (PNG or JPEG)')
```

Comment:
If you do the `accept` prop like above, this can be simplified a lot, or moved to the file upload handlers. Mentioning it there :)
You might even be able to lose the useState :)